pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -89,7 +103,7 @@ func New(imageFsInfoProvider ImageFsInfoProvider, rootPath string, usingLegacySt
 	// collect metrics for all cgroups
 	rawContainerCgroupPathPrefixWhiteList := []string{"/"}
 	// Create and start the cAdvisor container manager.
-	m, err := manager.New(memory.New(statsCacheDuration, nil), sysFs, maxHousekeepingInterval, allowDynamicHousekeeping, includedMetrics, http.DefaultClient, rawContainerCgroupPathPrefixWhiteList)
+	m, err := newCadvisor(memory.New(statsCacheDuration, nil), sysFs, maxHousekeepingInterval, allowDynamicHousekeeping, includedMetrics, http.DefaultClient, rawContainerCgroupPathPrefixWhiteList)
```

Comment:
talked with @dims about an approach that would avoid needing to change this signature and keep us (and all callers) from having to copy out initialization code

will open a PR to cadvisor and sync up there

Reply:
opened https://github.com/google/cadvisor/pull/2217, keeping all the collector initialization contained to `manager.New()` ... testing with CI in https://github.com/kubernetes/kubernetes/pull/76355